ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+---------------------------------------------------
+
+_page 5 sur 15_ 
+
+<table>
+<tr>
+<td>Principes</td>
+<td>→</td>
+<td>Pratique</td>
+<td>→</td>
+<td>Résultats</td>
+</tr>
+<tr>
+<td>
+• Échanges ouverts<br/> • Participation<br/> • Publier tôt + souvent<br/> •
+Méritocracie<br/> • Communauté
+</td>
+<td>→<br/>→<br/>→</td>
+<td>
+• Transparence avec les usagers internes et les autres parties prenantes <br/> •
+Implication des usagers<br/> • Récolter du feedback et s'adapter de manière
+itérative<br/> • Imaginer avec les usagers<br/> • Construire de la confiance et du respect via la collaboration 
+</td>
+<td>→<br/>→<br/>→</td>
+<td>
+• Adhésion des usagers<br/> • Adoption plus rapide et plus forte<br/> • Les
+meilleures idées gagnent<br/> • Moins de bugs, de problèmes et de impacts
+imprévus <br/> • Plus grand engagement des collaborateurs <br/> • Décisions
+alignées sur le stratégie et la culture
+</td>
+</tr>
+</table>
+
+<br/>
+
+On ne peut pas plaire à tout le monde. <a name="cant-please-everyone"></a>
+----------------------------------------------------
+
+**Mais qu'on prend une décision ouverte, les gens se disent...**  
+
+_page 6 sur 15_ 
+
+  - Je comprend pouquoi la décision a été prise et comment elle s'aligne avec la
+    stratégie, les buts et la mission de Red Hat.
+  - Les pré-requis de l'entreprise, les critères de recherche et d'évaluation
+    étaient visibles.
+  - Le processus de décision était inclusif et transparent.
+  - Même si je n'ai pas pris la décision, je peux contribuer au processus.
+  - Je ne suis peut-être pas d'accord avec la décision mais il est évident que
+    les personnes qui l'ont prise comprennent les valeurs et la culture de Red
+    Hat.
+  - Je suis peut-être déçu mais je ne suis pas surpris.
+  - Ma voix a été entendue et valorisée.
+
+<br/>
+
+Cadre de Décision Ouverte <a name="framework"></a>
+===========================================================================
+
+Phase ou activité: Conceptualiser, Définir, Imaginer <a name="ideation"></a>
+---------------
+
+_page 7 sur 15_ 
+
+<table>
+<tr>
+<td width=34%><strong>Étapes pour devenir ouvert </strong></td>
+<td width=36%><strong>Questions à poser</strong></td>
+<td width=30%><strong>Déclencheurs de flamewar</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3>
+<strong>Diriger avec transparence</strong><br/>
+ • Publier un énoncé du problème et des approches possibles<br/><br/>
+ • Identifer les aspects du projet ou de la décision qui ne peuvent pas être
+ouvert<br/><br/>
+ • Publier votre processus de conception<br/><br/>
+
+<strong>Batir une diversité de points de vue + un environement inclusif<strong><br/>
+ • Solliciter les usagers internes et les parties prenantes 
+très tôt, en particulier ceux qui pourraient ne pas être d'accord.<br/><br/>
+ • Chercher des perspectives variées et sous-représentées (emplacement
+géographique, ethnie, départements, niveau hiérarchique, genre, age, etc. )<br/><br/>
```

Comment:
I think that since "ethnie" and all the others are singular, we should keep "département" as singular too.